src/commands/createNewProject/PythonProjectCreator.ts

```diff
@@ -191,3 +195,24 @@ export class PythonProjectCreator extends ScriptProjectCreatorBase {
         }
     }
 }
+
+/**
+ * Returns undefined if valid or an error message if not
+ */
+async function validatePythonAlias(pyAlias: PythonAlias): Promise<string | undefined> {
+    try {
+        const result: cpUtils.ICommandResult = await cpUtils.tryExecuteCommand(undefined /*don't display output*/, undefined /*default to cwd*/, `${pyAlias} --version`);
+        if (result.code !== 0) {
+            return localize('failValidate', 'Failed to validate version:{1}{2}', pyAlias, os.EOL, result.cmdOutputIncludingStderr);
```

Comment:
You missing an arg?

Reply:
Cleaned up the code. Screenshot above is how I intended the message to look